docs/TOC.yml

```diff
@@ -5,6 +5,8 @@
       href: what-is-maui.md
     - name: Supported platforms
       href: supported-platforms.md
+    - name: Troubleshoot common problems
```

Comment:
Maybe just call it Troubleshooting?

Reply:
Personally, I'd also pub this at the bottom of the TOC (experience tells me this will eventually turn into an expandable node with multiple docs under it).